test/sqlsmith/mzcompose.py

```diff
@@ -0,0 +1,175 @@
+# considered product bugs, but SQLsmith generating bad queries. Use ci-regexp
+# in Github issues for actual product bugs.
+known_errors = [
+    "no connection to the server",  # Expected AFTER a crash, the query before this is interesting, not the ones after
+    "failed: Connection refused",  # Expected AFTER a crash, the query before this is interesting, not the ones after
+    'could not translate host name "materialized" to address: Name or service not known',  # Expected AFTER a crash, the query before this is interesting, not the ones after
+    "violates not-null constraint",
+    "division by zero",
+    "operator does not exist",  # For list types
+    "more than one record produced in subquery",
+    "invalid range bound flags",
+    "argument list must have even number of elements",
+    "array_agg on arrays not yet supported",
+    "sum(interval) not yet supported",
+    "mz_row_size requires a record type",
+    "invalid input syntax for type jsonb",
+    "invalid regular expression",
+    "aggregate functions are not allowed in",
+    "invalid input syntax for type date",
+    "invalid escape string",
+    "invalid hash algorithm",
+    "nested aggregate functions are not allowed",
+    "is defined for numbers greater than or equal to",
+    "is not defined for zero",
+    "is not defined for negative numbers",
+    "requested character too large for encoding",
+    "internal error: unrecognized configuration parameter",
+    "invalid encoding name",
+    "invalid time zone",
+    "value out of range: overflow",
+    "LIKE pattern exceeds maximum length",
+    "negative substring length not allowed",
+    "cannot take square root of a negative number",
+    "timestamp units not yet supported",
+    "step size cannot equal zero",
+    "stride must be greater than zero",
+    "timestamp out of range",
+    "unterminated escape sequence in LIKE",
+    "null character not permitted",
+    "is defined for numbers between",
+    "field position must be greater than zero",
+    "' not recognized",  # Expected, see https://github.com/MaterializeInc/materialize/issues/17981
+    "must appear in the GROUP BY clause or be used in an aggregate function",
+    "Expected joined table, found",  # Should fix for multi table join
+    "Expected ON, or USING after JOIN, found",  # Should fix for multi table join
+    "but expression is of type",  # Should fix, but only happens rarely
+    "coalesce could not convert type map",  # Should fix, but only happens rarely
+    "operator does not exist: map",  # Should fix, but only happens rarely
+    "result exceeds max size of",  # Seems expected with huge queries
+    "expected expression, but found reserved keyword",  # Should fix, but only happens rarely with subqueries
+    "Expected right parenthesis, found left parenthesis",  # Should fix, but only happens rarely with cast+coalesce
+    "invalid selection: operation may only refer to user-defined tables",  # Seems expected
```

Comment:
This I think happens with `INSERT ... SELECT` where the `SELECT` references something that is not a user table.